doc/src/manual/code-loading.md

```diff
@@ -151,26 +157,35 @@ and gets `2d15fe94-a1f7-436c-a4d8-07a9a496e01c`, which indicates that in the con
 
 What happens if `import Zebra` is evaluated in the main `App` code base? Since `Zebra` does not appear in the project file, the import will fail even though `Zebra` *does* appear in the manifest file. Moreover, if `import Zebra` occurs in the public `Priv` package—the one with UUID `2d15fe94-a1f7-436c-a4d8-07a9a496e01c`—then that would also fail since that `Priv` package has no declared dependencies in the manifest file and therefore cannot load any packages. The `Zebra` package can only be loaded by packages for which it appear as an explicit dependency in the manifest file: the  `Pub` package and one of the `Priv` packages.
 
-**The paths map** of a project environment is also determined by the manifest file if present and is empty if there is no manifest. The path of a package `uuid` named `X` is determined by these two rules:
+**The paths map** of a project environment is also determined by searching through the manifest files that are present in the load path. The path to the folder for a given package, identified by `uuid` and `Name`, is determined by these rules (in order):
```

Comment:
```suggestion
**The paths map** of a project environment is also determined by the manifest file. If there is no manifest file, then the paths map of the environment is empty. The path of a package `uuid` named `X` is determined by these two rules:
```

Reply:
> If there is no manifest file, then the paths map of the environment is empty

If there's no manifest (or an empty manifest), the path map may still contain a single entry (the path to the project environment itself)